front/app/utils/testUtils/dragAndDrop.ts

```diff
@@ -0,0 +1,13 @@
+import { fireEvent } from './rtl';
+
+export default function dragAndDrop(
```

Comment:
I'll share this one in the next FE CDM.

Reply:
You read my mind! 😄 Normally I would say we should be more conservative with new utilities but this is such a pain that it's totally worth having in our test utils.